packages/edit-site/src/components/global-styles/screen-root.js

```diff
@@ -98,6 +98,38 @@ function ScreenRoot() {
 					</NavigationButtonAsItem>
 				</ItemGroup>
 			</CardBody>
+
+			<CardDivider />
+
+			<CardBody>
+				<Spacer
+					as="p"
+					paddingTop={ 2 }
+					/*
+					 * 13px matches the text inset of the NavigationButton (12px padding, plus the width of the button's border).
+					 * This is an ad hoc override for this particular instance only and should be reconsidered before making into a pattern.
+					 */
+					paddingX="13px"
+					marginBottom={ 4 }
+				>
```

Comment:
have updated to the original comment rather than repeating the same comment twice with a note about revisiting this override if needed to be used wider than these two examples. Could be followed up regardless to tidy up if needed.